brain/CMakeLists.txt

```diff
@@ -8,16 +8,22 @@ source_group(\\ FILES CMakeLists.txt)
 
 set(BRAIN_PUBLIC_HEADERS
   circuit.h
-  morphology.h
   types.h
+  cell/morphology.h
+  cell/section.h
+  cell/soma.h
```

Comment:
# All: please vote:

Option 1: As above, except s/cell/neuron/g
Option 2:
```
morphology.h
morphology/section.h
morphology/soma.h
```

## Option 1 rationale: 

All functionalities wrt morphologies contained in a single namespace. Adding glial morphology would not collide.

## Option 2 rationale: 

This option mimics nested classes cleanly using namespaces.

Symmetry with brion::Morphology and in naming: for section, it's a morphology::Section. Maybe morphology should be renamed neuron/morphology here and in brion:: (tbd when glia is introduced: aka defer Option 1).